web-app/js/portal/ui/openlayers/control/SpatialConstraint.js

```diff
@@ -14,6 +14,13 @@ Portal.ui.openlayers.SpatialConstraintType = {
 Portal.ui.openlayers.control.SpatialConstraint = Ext.extend(OpenLayers.Control.DrawFeature, {
 
     MIN_AREA_PERCENT: 0.01,
+    errorStyle: {
+        fillOpacity:0.3,
```

Comment:
Wouldn't you want something like `'0.3px'`? Add a space here anyway after the colon.